### DISCUSSION TITLE
左上にコメント数やコメントへのリンクがほしい

### DISCUSSION THREAD

**Discussion by ko1nksm** [increments/qiita-discussions#905] [要望・アイデア・不具合]
記事の左上にコメント数やコメントへのリンクがほしいです。現在は記事の最後にだけ表示されていますが、記事が長いと記事の最後にたどり着けません。

![image](https://github.com/user-attachments/assets/8173a885-f2ca-4243-ac7e-66b1f25563c6)

**Comment by tanaka-asami415** [comment]
@ko1nksm 
ご意見ありがとうございます！

https://github.com/increments/qiita-discussions/discussions/234#discussioncomment-9056862 の修正の際に
上部にもリンクを付けることを検討しましたが、ジャンプ時に位置がズレてしまう問題があり、一旦現在の状態となっております🙇
将来的にはこの問題を解決して、上部にもリンクを付けたいと考えています。
